eng/Versions.props

```diff
@@ -151,7 +151,7 @@
     <XamarinMacCatalystWorkloadManifestVersion>14.3.100-ci.pr.gh10849.351</XamarinMacCatalystWorkloadManifestVersion>
     <XamarinMacOSWorkloadManifestVersion>11.1.100-ci.pr.gh10849.1259</XamarinMacOSWorkloadManifestVersion>
     <XamarinTvOSWorkloadManifestVersion>14.3.100-ci.pr.gh10849.1259</XamarinTvOSWorkloadManifestVersion>
-    <BlazorWorkloadManifestVersion>6.0.0-preview.3.21161.5</BlazorWorkloadManifestVersion>
+    <BlazorWorkloadManifestVersion>$(MicrosoftAspNetCoreAppRuntimePackageVersion)</BlazorWorkloadManifestVersion>
```

Comment:
Similar to https://github.com/dotnet/installer/blob/main/eng/Versions.props#L109